getpaid/utils.py

```diff
@@ -1,107 +1,10 @@
-# coding: utf8
-import sys
-from collections import OrderedDict
-from importlib import import_module
+import collections
 
-from django.conf import settings
-from django.contrib.sites.shortcuts import get_current_site
-from django.urls import reverse
-from django.utils import six
-from django.utils.six.moves.urllib.parse import parse_qsl
 
-if six.PY3:
-    unicode = str
-
-
-def import_backend_modules(submodule=None):
-    backends = getattr(settings, 'GETPAID_BACKENDS', [])
-    modules = {}
-    for backend_name in backends:
-        fqmn = backend_name
-        if submodule:
-            fqmn = '%s.%s' % (fqmn, submodule)
-        __import__(fqmn)
-        module = sys.modules[fqmn]
-        modules[backend_name] = module
-    return modules
-
-
-def get_backend_choices(currency=None):
-    """
-    Get active backends modules. Backend list can be filtered by
-    supporting given currency.
-    """
-    choices = []
-    backends_names = getattr(settings, 'GETPAID_BACKENDS', [])
-
-    for backend_name in backends_names:
-        backend = import_module(backend_name)
-        if currency:
-            if currency in backend.PaymentProcessor.BACKEND_ACCEPTED_CURRENCY:
-                choices.append(
-                    (backend_name, backend.PaymentProcessor.BACKEND_NAME)
-                )
+def update(d, u):
```

Comment:
A docstring would be really heplful. Also it could be called e.g. recursive_update